doc/user_guide/marks/geoshape.rst

```diff
@@ -0,0 +1,689 @@
+    gdf_comb = gpd.GeoDataFrame(source.join(us_states, on='id', rsuffix='_y'))
+
+    alt.Chart(gdf_comb).mark_geoshape().encode(
+        color=alt.Color('value:Q'),
+        facet=alt.Facet('variable:N', columns=3)
+    ).properties(
+        width=180,
+        height=130
+    ).resolve_scale('independent')
+
+For now, the following workaround can be adopted to facet a map, manually filter the 
+data in pandas, and create a small multiples chart via concatenation. For example:
+
+.. altair-plot::
+
+    alt.concat(
+        *(
+            alt.Chart(gdf_comb[gdf_comb.variable == var], title=var)
+            .mark_geoshape()
+            .encode(
+                color=alt.Color(
+                    "value:Q", legend=alt.Legend(orient="bottom", direction="horizontal")
+                )
+            )
+            .properties(width=180, height=130)
+            for var in gdf_comb.variable.unique()
+        ),
+        columns=3
+    ).resolve_scale(color="independent")
+
+
+
+Interaction
+~~~~~~~~~~~
+Often a map does not come alone, but is used in combination with another chart. 
+Here we provide an example of an interactive visualization of a bar chart and a map.
+
+The data shows the states of the US in combination with a bar chart showing the 15 most 
+populous states. Using an ``alt.selection_point()`` we define a selection parameter that connects to our left-mouseclick.
+
+.. altair-plot::
+
+    import altair as alt
+    from vega_datasets import data
+    import geopandas as gpd
+
+    # load the data
+    us_states = gpd.read_file(data.us_10m.url, driver='TopoJSON', layer='states')
+    us_population = data.population_engineers_hurricanes()[['state', 'id', 'population']]
+
+    # define a pointer selection
+    click_state = alt.selection_point(fields=['state'])
+
+    # create a chloropleth map using a lookup transform 
+    # define a condition on the opacity encoding depending on the selection
+    chloropleth = alt.Chart(us_states).mark_geoshape().transform_lookup(
+        lookup='id',
+        from_=alt.LookupData(us_population, 'id', ['population', 'state'])
+    ).encode(
+        color='population:Q',
+        opacity=alt.condition(click_state, alt.value(1), alt.value(0.2)),
+        tooltip=['state:N', 'population:Q']
+    ).project(type='albersUsa').add_params(click_state)
+
+    # create a bar chart with a similar condition on the opacity encoding.
+    bars = alt.Chart(
+        us_population.nlargest(15, 'population'),
+        title='Top 15 states by population').mark_bar(
+    ).encode(
+        x='population',
+        opacity=alt.condition(click_state, alt.value(1), alt.value(0.2)),
+        color='population',
+        y=alt.Y('state', sort='-x')
+    ).add_params(click_state)
+
+
+    chloropleth & bars
+
+The interaction is two-directional. If you click (shift-click for multi-selection) on a geometry or bar the selection receive an ``opacity`` of ``1`` and the remaining an ``opacity`` of ``0.2``.
+
+Expression
```

Comment:
So cooooooool!

Reply:
It is based on your example as defined here https://stackoverflow.com/questions/63751130/altair-choropleth-map-color-highlight-based-on-line-chart-selection/65948144#65948144. But definitely cool😃.

Reply:
@mattijn My comment was about your coooool example with the rotating globe and the earthquakes =) That one is really neat!